src/style.css

```
@@ -0,0 +1,80 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "avenirFont";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "avenirFont";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+.container {
+  width: 100%;
+  height: 70px;
+  margin: 20px 0;
+}
+
+.containerS {
+  height: 42px;
+}
+
+.inputField {
+  border-style: none;
+  outline: none;
+  width: 100%;
+  line-height: 70px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  font-weight: 300;
+  color: #3d4e61;
+  font-family: "avenirFont", sans-serif;
+}
+
+.inputFieldS {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+}
+
+.inputField:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.inputField:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.inputContainer {
+  height: 100%;
+  position: relative;
+}
+
+.icon {
+  position: absolute;
+  z-index: 1;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.iconS {
```

Use definitive class names
```suggestion
.iconSmall {
```